contrib/kind.sh

```diff
@@ -322,6 +344,12 @@ parse_args() {
             -mne | --multi-network-enable )     shift
                                                 ENABLE_MULTI_NET=true
                                                 ;;
+            -ic | --interconnect-enable )       shift
+                                                OVN_INTERCONNECT_ENABLE=true
+                                                ;;
+            -lz | --enable-local-zone )       shift
+                                                OVN_ENABLE_LOCAL_ZONE=true
+                                                ;;
```

Comment:
Don't shift for these two parameters as they carry no value

Reply:
Ack.  I missed this.  I'll update in the next push